Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -50,19 +50,33 @@
 require "missing_formula"
 require "digest"
 
+require_relative "../cli_parser"
```

Comment:
Would be preferable to use a regular `require` here.

Reply:
Addressed